### ISSUE TITLE
Component overflows when used inside a bootstrap column

### ISSUE THREAD

**Issue by xabikos** [furqanZafar/react-selectize#61]
I see that in the css the width is set to 300px. This has a result the component to overflows when is used inside a bootstrap column as in the following image

![image](https://cloud.githubusercontent.com/assets/487416/13951486/97d71e32-f031-11e5-8595-4e15ca0ce2da.png)

Is this a choice or I miss something?

**Comment by furqanZafar** [comment]
not a deliberate choice, will fix it in the next patch

**Comment by apucacao** [comment]
Hey @furqanZafar, are you still planning on removing the hardcoded width on the root node?

**Comment by furqanZafar** [comment]
yep, open to PR though, actually planning a rewrite in babel ES6 & adopting BEM for css

**Comment by xabikos** [comment]
I would happily open a pull request but as I mentioned in #69 I am not familiar with LiveScript at all

**Comment by gsanta** [comment]
Until it is resolved, you can overwrite this property with 'width: auto'. That worked for me.
